core/protos/o2control.proto

```diff
@@ -186,6 +186,7 @@ message EnvironmentInfo {
     map<string, string> userVars = 9;
     int32 numberOfFlps = 10;
     repeated string includedDetectors = 11;
+    string rootRoleDescription = 12;
```

Comment:
I'd just call it `description`. We only read the description from the root role anyway, the field is ignored in other places.